selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -185,6 +199,7 @@ public CodenvyEditor(
     String JAVA_DOC_POPUP = "//div[@class='gwt-PopupPanel']//iframe";
     String AUTOCOMPLETE_PROPOSAL_JAVA_DOC_POPUP =
         "//div//iframe[contains(@src, 'api/java/code-assist/compute/info?')]";
+    String HIGHLIGHT_ITEM = "//li[@selected='true']//span[text()='%s']";
```

Comment:
It's a pattern. It's better to express this moment in field name to avoid confusion.

Reply:
And what is your version? "HIGHLIGHT_ITEM_PATTERN" ?

Reply:
Yes, it is.